lib/src/builder/parsing/declarations_from_members.dart

```diff
@@ -360,6 +402,25 @@ Iterable<BoilerplateDeclaration> getBoilerplateDeclarations(
   }
 }
 
+// Returns whether or not the props config argument of `uiFunction` is `null`.
+bool _hasConfigArg(BoilerplateFactory factory) {
+  final rightHandSide = factory.node.variables.firstInitializer;
+  final args = (rightHandSide as MethodInvocation).argumentList.arguments;
+  if (args == null || args.length < 2) return false;
+  return args[1] is! NullLiteral;
+}
+
+// Returns the name of the props for [factory] based on typing arguments in the `uiFunction` declaration.
+String _getPropsNameFromFunctionComponent(BoilerplateFactory factory) {
```

Comment:
This should be a getter, similar to `BoilerplateComponent.propsGenericArg`

Reply:
That's so much nicer!